simulator/tabletop.py

```diff
@@ -74,3 +89,26 @@ def start_app_on_all(self, app_type):
   def run(self):
     """ Runs the tabletop simulation indefinitely. """
     self.__canvas.wait_for_events()
+  
+  def draw_grid(self):
+    """ Creates Line objects for grid, returns the list of line objects in grid."""
+    grid = []
+    window_x, window_y = self.__canvas.get_window_size()
+    i = 0 
+    while i < window_x:
+      line = Line(self.__canvas,( i, 0), (i, window_y))
+      grid.append(line)
+      i += 200
+    j = 0
+    while j < window_y:
+      line = Line(self.__canvas,(0, j), (window_x, j))
+      grid.append(line)
+      j += 200
+    return grid
+    
+  def clear_grid(self):
+    """ Removes line ID's from grid, deletes the lines from canvas."""
+    for i in range(len(self.__grid)):
+      line = self.__grid.pop()
```

Comment:
Probably slightly more efficient and readable to clear the entire list at the end of the loop.